src/NDItk/MultigroupTable/test/MultigroupTable.test.cpp

```diff
@@ -35,10 +35,13 @@ SCENARIO( "MultigroupTable" ) {
       multigroup::FluxWeights weights( { 0.1, 0.2, 0.25, 0.05, 0.15, 0.04, 0.06 } );
       multigroup::ReactionCrossSections xs( { { 2, 0.0, { 10., 20., 30., 40., 50., 60., 70. } },
                                               { 16, 1.1234567, { 1., 2., 3., 4., 5., 6., 7. } } } );
+      multigroup::AverageFissionEnergyRelease release( 202.827, 181.238898, 4.827645,
```

Comment:
Is it okay that this stuff is in a different order than in L150?

Reply:
Yes, since they get rearranged under the hood. The order I used here is:
```
 *  @param[in] total       the total energy release (including delayed particles
 *                         and neutrinos)
 *  @param[in] prompt      the recoverable energy release (total minus delayed
 *                         particles and neutrinos)
 *  @param[in] neutrons    the energy release through prompt neutrons
 *  @param[in] gammas      the energy release through prompt gammas
 *  @param[in] betas       the energy release through delayed betas
 *  @param[in] fragments   the kinetic energy of fission fragments
```

Reply:
But the order is different if we read from a file versus input it this way?

Reply:
The order under the hood is the same as in the file

Reply:
Let me rephrase: the data given to the constructor gets rearranged to the order as defined by the NDI format.